tensorpipe/transport/shm/socket.h

```diff
@@ -210,13 +210,29 @@ class Socket final : public Fd, public std::enable_shared_from_this<Socket> {
   // Send file descriptor.
   template <typename... Fds>
   Error sendFds(const Fds&... fds) {
-    return sendFdsToSocket(fd_, fds...);
+    struct {
+    } nothing;
```

Comment:
I was under the impression that the payload must contain at least one byte, but I cannot find any confirmation of that at the moment. Does that ring a bell? Do you know why it was one byte before?